src/slicer.cpp

```diff
@@ -780,6 +780,9 @@ void SlicerLayer::makePolygons(const Mesh* mesh, bool keep_none_closed, bool ext
     polygons.removeDegenerateVerts(); // remove verts connected to overlapping line segments
 
     int xy_offset = mesh->getSettingInMicrons("xy_offset");
+    if(is_initial_layer)
+        xy_offset = mesh->getSettingInMicrons("xy_offset_layer_0");
+
```

Comment:
I implemented it in a way that it is in line with other "Initial Layer" settings. "Initial Layer Height" is not a delta to "Layer Height", and neither is "Printing Temperature Initial Layer". So in my opinion it would be counter-intuitive if this one new setting were.